src/neo/SmartContract/Native/Designate/DesignateContract.cs

```diff
@@ -28,16 +29,22 @@ internal override void Initialize(ApplicationEngine engine)
         {
             foreach (byte role in Enum.GetValues(typeof(Role)))
             {
-                engine.Snapshot.Storages.Add(CreateStorageKey(role), new StorageItem(new NodeList()));
+                engine.Snapshot.Storages.Add(CreateStorageKey(role).AddBigEndian(0u), new StorageItem(new NodeList()));
```

Comment:
Do we need to set up initial members? If it is not found in `FindRange()`, then just return empty. This eliminates the need for `Initialize()`.

Reply:
Done